app/scripts/modules/amazon/src/svg.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module '*.svg' {
```

Comment:
I would _really_ like to find a good way of doing this without type declarations. [The docs have a guide on how to create a custom template string that actually creates TS code for the autogenerated components](https://react-svgr.com/docs/typescript/), but when I tried that it was a total mess because the generated code has to then also pass through the `ts-loader` / `thread-loader` etc. etc. path which kept leading to obscure and weird errors. Using a custom template also means we can't make use of some of the really clever optimizations the default template has like creating `const`s for all the paths inside the SVG in order to reuse the same JSX elements across renders.

I think what i really want is a way to do this module declaration in a spot where it doesn't have to be repeated for every single module.